sciences-po-ecole-doctorale-note-french.csl

```diff
@@ -0,0 +1,423 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="note" version="1.0" page-range-format="expanded" demote-non-dropping-particle="sort-only" default-locale="fr-FR">
+  <info>
+    <title>Sciences Po - Ecole doctorale (note, French)</title>
```

Comment:
Not critical, but generally we include diacrits in the title field (at least where we notice them misssing...) and I believe this would be `École`, right? (searching for "ecole" on Zotero.org/styles would find both.

Reply:
ok, many thanks
i just put École in the title field.
Best regards,
Anna